src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -681,19 +729,22 @@ public void testFileIdUpdateMethods() throws Exception {
   private List<String> readIds(Dataset dataset) throws Exception {
     BigQueryProject bigQueryDatasetProject =
         TestUtils.bigQueryProjectForDatasetName(datasetDao, dataset.getName());
+    Column fileColumn = new Column().name("file");
     List<String> ids =
         new ArrayList<>(
             queryForColumn(
                 BigQueryPdao.prefixName(dataset.getName()),
                 getTable(dataset, "file").getName(),
-                new Column().name("file"),
-                bigQueryDatasetProject));
+                fileColumn,
+                bigQueryDatasetProject,
+                fileColumn));
     List<List<String>> idsFromArray =
         queryForColumn(
             BigQueryPdao.prefixName(dataset.getName()),
             getTable(dataset, "file").getName(),
             new Column().name("file_a").arrayOf(true),
-            bigQueryDatasetProject);
+            bigQueryDatasetProject,
+            fileColumn);
```

Comment:
this one can't be ordered by the column ("file_a") because it contains an array, so I needed to add a separate parameter for orderBy